lib/agent0/agent0/hyperdrive/exec/execute_agent_trades.py

```diff
@@ -45,14 +44,19 @@ async def async_execute_single_agent_trade(
         The HyperdriveAgent that is conducting the trade
     hyperdrive : HyperdriveInterface
         The Hyperdrive API interface object
+    liquidate: bool
+        If set, will ignore all policy settings and liquidate all open positions
 
     Returns
     -------
     list[TradeResult]
         Returns a list of TradeResult objects, one for each trade made by the agent
         TradeResult handles any information about the trade, as well as any errors that the trade resulted in
     """
-    trades: list[types.Trade[HyperdriveMarketAction]] = agent.get_trades(interface=hyperdrive)
+    if liquidate:
+        trades: list[types.Trade[HyperdriveMarketAction]] = agent.liquidation_trades()
```

Comment:
should we call the func `get_liquidation_trades` to be consistent with `get_trades`?